tests/ert_tests/ert3/console/integration/test_cli.py

```diff
@@ -666,6 +681,75 @@ def test_cli_local_test_run_specific_realization(tmpdir):
             assert pathlib.Path(f"local-test-run-{run_id}/output.json").exists()
 
 
+def test_cli_local_test_run_storageinput(tmpdir):
```

Comment:
Maybe this entire thing can be an example to be tested in line with the others?

Reply:
As in https://github.com/equinor/ert/pull/3077/commits/f62c5709f899adb91e7db94898208a9030ed1e39 ?